repetitions/game.rb

```diff
@@ -0,0 +1,40 @@
+def info_request(question)
+  puts question
+  Integer(gets.chomp)
+end
+
+def number_pick(input)
+  if input == 1
```

Comment:
tutaj najprościej by było zastosować wzorzec projektowy dependency injection:
```
def number_pick(input, rand_object=Kernel)
  if input == 1
    level = rand_object.rand(10)
  (...)
end
```

i wtedy w testach:
```
require 'ostruct'

number_pick(input, OpenStruct.new(rand: 10)
```
wtedy zamockujesz sobie w ten sposób wywołanie rand, że zawsze będzie zwracać to samo.

Można o tym doczytać tutaj:

* http://sporto.github.io/blog/2013/09/25/simple-dependency-injection/
* https://en.wikipedia.org/wiki/Dependency_injection